apps/epic-web/src/components/cta/resource-cta.tsx

```diff
@@ -294,9 +294,16 @@ const ActivePromotionCTA = ({
     return null
   }
 
+  // TODO: This is a temporary fix to check if the product is a megabundle,
+  // it would be better to have a flag in the product itself!
+  const IS_MEGABUNDLE =
+    currentActivePromotion.product.slug.includes('megabundle')
+
```

Comment:
we don't handle multiple instructors, etc. yet

<img width="500" alt="image" src="https://github.com/user-attachments/assets/3ca7e09f-b324-49e6-a759-390b9d8ff388">